_inc/idc-notice.js

```diff
@@ -15,21 +15,31 @@
 		fixJetpackConnection();
 	} );
 
+	function disableDopsButtons() {
+		$( '.dops-button' ).prop( 'disabled', true );
+	}
+
+	function enableDopsButtons() {
+		$( '.dops-button' ).prop( 'disabled', false );
+	}
+
 	function confirmSafeMode() {
-		var route = restRoot + 'jetpack/v4/site';
+		var route = restRoot + 'jetpack/v4/identity-crisis/confirm-safe-mode';
+		disableDopsButtons();
 		$.ajax( {
-			method: 'GET',
+			method: 'POST',
 			beforeSend : function ( xhr ) {
 				xhr.setRequestHeader( 'X-WP-Nonce', restNonce );
 			},
 			url: route,
 			data: {},
-			success: function( response ){
+			success: function(){
 				$( '.jp-idc-notice' ).hide();
-				alert( JSON.stringify( response, null, 4 ) );
+				alert( 'Safe mode confirmed.' );
 			},
 			error: function( response ) {
 				console.log( response.responseText );
```

Comment:
Since we're so close to having the "Confirm" action done, should we go ahead and remove this `console` as well?